hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/ui/UIAutoConfiguration.java

```diff
@@ -37,4 +36,14 @@ public DistributedResourceBundleMessageSource messageSource() {
         return new DistributedResourceBundleMessageSource();
     }
 
+    /**
+     * A event provider bean which hold the supported events for the UI.
+     * 
+     * @return the provider bean
+     */
+    @Bean
```

Comment:
Why we adding the bean into the `AutoConfiguration` if you create the `HawkbitEventProvider` by our own with `new` in the constructor of the example app? I guess we don't need to provide the autoconfiguration for this, do we? Or otherwise we should annotate it with `@ConditionalMissingBean`